data-sources/atlassian-jira/config.yml

```
@@ -0,0 +1,31 @@
+id: atlassian-jira
+
+displayName: Atlassian JIRA
+
+description: |
+  Jira is a proprietary issue tracking product developed by Atlassian that allows bug tracking and agile project management. You can Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
+
+  [Check out our documentation](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#jira) to set up a JIRA notification channel and provide fast and consistent ways for the right personnel to be notified about incidents
+
+logo: logo.png
+
+install:
+  primary:
+    nerdlet: nr1-install-newrelic.quickstart-installation-plan
+    nerdletState:
+      quickstart_id: a99b22a5-5666-4096-b239-8f315700fde0
+  requiresAccount: true
+  secondary:
```

The API calls this field `fallback` instead of `secondary`, since we're converting the others to match up, let's change this as well.
```suggestion
  fallback:
```

---

updated